README.md

```diff
@@ -2,102 +2,102 @@
-
-
-## Client Drivers (community)
-
-- [`client-elixir`](https://github.com/taxfix/grakn_elixir)
-- [`Grakn.Net`](https://github.com/WalternativE/Grakn.Net)
-- [`TypeDBClient.jl`](https://github.com/Humans-of-Julia/TypeDBClient.jl)
-
-
-## IDE and editors support
-
-- [`typeql-plugin-emacs`](https://github.com/typedb-osi/typeql-plugin-emacs) - Major mode that provides syntax highlighting and basic completion for TypeQL.
-- [`typeql-plugin-vscode`](https://github.com/typedb-osi/typeql-plugin-vscode) - Visual Studio Code support for TypeQL.
-- [`typeql-plugin-jetbrains`](https://github.com/typedb-osi/typeql-plugin-jetbrains) - JetBrains support for TypeQL.
-- [`typeql-plugin-atom`](https://github.com/typedb-osi/typeql-plugin-atom) - Atom's syntax highlighting for TypeQL.
-
-
 ## Open source projects using TypeDB
 
-- [`TypeDB OSI`](https://github.com/typedb-osi) - TypeDB OSI Ecosystem comprises all resources and technologies to be used with TypeDB.
-- [`TypeDB Loader`](https://github.com/bayer-science-for-a-better-life/grami) - Formerly "GraMi", an open-source data migrator for loading data into TypeDB at scale.
+- [`KGLIB - Machine Learning with TypeDB`](https://github.com/vaticle/kglib) - KGLIB provides tools to enable machine learning with TypeDB.
+- [`TypeDB Studio - IDE for TypeDB`](https://github.com/vaticle/typedb-studio) - TypeDB Studio is an Integrated Development Environment for TypeDB for performing knowledge engineering.
+- [`TypeDB OSI - Cyber Threat Intelligence`](https://github.com/typedb-osi/typedb-cti) - TypeDB CTI is an open source threat intelligence platform enabling threat intel professionals to bring together their disparate CTI information into one database 
+and find new insights about cyber threats.
+- [`TypeDB OSI - Biomedical Knowledge Graph`](https://github.com/typedb-osi/typedb-bio) - TypeDB Bio is an open source biomedical knowledge graph providing biomedical researchers an intuitive way to query interconnected and heterogeneous biomedical 
+data in one single place.
+- [`TypeDB Loader`](https://github.com/typedb-osi/typedb-loader) - An open-source data migrator for loading data into TypeDB at scale.
 - [`Source++`](https://github.com/sourceplusplus/Assistant) - Open-source automated programming assistant with runtime-aware and context-aware functionality.
 - [`Rasa`](https://github.com/RasaHQ/rasa) - Open-source machine learning framework to automate text-and voice-based conversations.
-- [`Grakn Mythos`](https://github.com/BFergerson/grakn-mythos) - Grakn-powered application for creating and sharing knowledge graphs.
 
 ## Conferences
 
-- [`Cosmos`](https://grakncosmos.com/)
+- [`Cosmos`](https://vaticle.com/conferences/typedb-cosmos-2022)
 - [`Orbit`](https://www.youtube.com/playlist?list=PLtEF8_xCPklY3P5NLSQb1SyIYLhQssxfY)
 
 ## Documentation
 
-- [`Documentation for mainline version`](https://docs.vaticle.com/)
-- [`Documentation for alpha version`](https://https://docs.vaticle.com/)
```

Comment:
I'm not sure what this second link to documentation was ever pointing at, but now we point to some great jumping off points in our documentation.